File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--simple-data-binding-in-vsto-add-in-project_4.cs
            extendedDocument.Paragraphs[1].Range.InsertParagraphBefore();
            extendedDocument.Paragraphs[1].Range.InsertParagraphBefore();
            extendedDocument.Paragraphs[1].Range.Text = 
                "The companies listed in the AdventureWorksLT database:   \n";
            extendedDocument.Paragraphs[2].Range.Text = "  "; 

            Word.Range range1 = extendedDocument.Paragraphs[2].Range.Characters.First;
            Word.Range range2 = extendedDocument.Paragraphs[2].Range.Characters.Last;
            Word.Range range3 = extendedDocument.Paragraphs[1].Range.Characters.Last;

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--simple-data-binding-in-vsto-add-in-project_4.vb
        extendedDocument.Paragraphs(1).Range.InsertParagraphBefore()
        extendedDocument.Paragraphs(1).Range.InsertParagraphBefore()
        extendedDocument.Paragraphs(1).Range.Text = "The companies listed in the AdventureWorksLT database:   "
        extendedDocument.Paragraphs(2).Range.Text = "  "

        Dim range1 As Word.Range = extendedDocument.Paragraphs(2).Range.Characters.First
        Dim range2 As Word.Range = extendedDocument.Paragraphs(2).Range.Characters.Last
        Dim range3 As Word.Range = extendedDocument.Paragraphs(1).Range.Characters.Last